server/controllers/conversation.js

```diff
@@ -0,0 +1,67 @@
+const Conversation = require("../models/Conversation");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /conversations
+// @desc Create new conversation
+// @access Public
+exports.createConversation = asyncHandler(async (req, res, next) => {
+  const { dogOwnerId, dogSitterId } = req.body;
+
+  try {
+    const conversation = await Conversation.create({
+      dogOwnerId,
+      dogSitterId
+    });
+    if (conversation)
+      res.status(201).json({ success: "Successfully created conversation" });
+    else
```

Comment:
We also don't need this `else` clause - if the Conversation document creation has failed, the rejection will be handled by the catch block

Ditto where applicable